platform-sdk/platform-apps/tests/AddressBookTestingTool/src/main/java/com/swirlds/demo/addressbook/AddressBookTestingToolState.java

```diff
@@ -223,7 +223,11 @@ public void handleConsensusRound(@NonNull final Round round, @NonNull final Plat
      * @param transaction the transaction to apply
      */
     private void handleTransaction(@NonNull final ConsensusTransaction transaction) {
-        final int delta = ByteUtils.byteArrayToInt(transaction.getContents(), 0);
+        if (transaction.isSystem()) {
```

Comment:
You're ignoring system transactions in several places, but it's not immediately clear from the code why that should be the case

Reply:
when the whole project is done, we are supposed to expose system transactions to apps. Hedere will write them to the block stream. In preparation for that, the demo apps will ignore these